Dockerfiles/cluster-agent/README.md

```diff
@@ -0,0 +1,33 @@
+# Cluster Agent 6 docker image
+
+This is how the official Datadog Cluster Agent 6 (aka DCA) image available [here](https://hub.docker.com/r/datadog/cluster-agent-dev/) is built.
```

Comment:
we need a `cluster-agent` repo as well. `cluster-agent-dev` is not the official repo, we'll use it for dev/testing.

Reply:
yeah, I figured we'd update this when we release a functional MVP. Do I still point to an empty  repo in the meantime ?